kura/org.eclipse.kura.driver.opcua.provider/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,10 @@ Bundle-Vendor: Eclipse Kura
 Bundle-License: Eclipse Public License v1.0
 Bundle-Category: Asset-Driver Management
 Bundle-ActivationPolicy: lazy
-Import-Package: org.eclipse.kura;version="[1.2,2.0)",
+Import-Package: javax.security.cert,
```

Comment:
remove `org.eclipse.osgi.service.urlconversion` from import-package as it is not used any more. I used it for dynamic classloading.